cmd/cli/types.go

```diff
@@ -64,6 +64,13 @@ type InvokeRequest struct {
 	ResouceLimits []ResourceLimit   `json:"resource_limits"`
 }
 
+type GasPrice struct {
+	CpuRate  int64 `json:"cpu_rate"`
```

Comment:
It's better to use uint64 to ensure no negative value.

Reply:
In order to align  with  the type having existed already, checking the value so that ensuring  that the parameters should not be negative instead.
Please read the latest pull request and this request has been expired.